<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>
<title>

Results

</title>
<meta content="text/html;charset=utf-8" http-equiv="Content-Type"/>
<script type="text/javascript">

function toggle(item) {

obj=document.getElementById(item);

visible=(obj.style.display!="none")

key=document.getElementById("ctrl_" + item);

if (visible) {

    obj.style.display="none";

key.innerHTML="[+] Show genes";

}

else {

obj.style.display="block";

key.innerHTML="[-] Hide genes";

}

}

</script>
</head>
<body>
<p><center><b>
</b></center></p>
<table align="center" border="1" id="tab:pval_legend" width="80%">
<caption><em>P-value color scale</em></caption>
<tr>
<td align="center" width="140"> &gt; 10<sup>-3</sup></td>
<td align="center" bgcolor="#FFFFCC" width="140"> 10<sup>-3</sup> to 10<sup>-5</sup></td>
<td align="center" bgcolor="#FFCC66" width="140"> 10<sup>-5</sup> to 10<sup>-7</sup></td>
<td align="center" bgcolor="#FF9900" width="140"> 10<sup>-7</sup> to 10<sup>-9</sup></td>
<td align="center" bgcolor="#FF3300" width="140"> &lt; 10<sup>-9</sup></td>
</tr>
</table>
<map name="goPathImage">
<area alt="" coords="208,197,299,245" href="#0044424" shape="RECT"/>
<area alt="" coords="151,294,241,357" href="#0044446" shape="RECT"/>
<area alt="" coords="12,301,127,349" href="#0044815" shape="RECT"/>
<area alt="" coords="95,501,180,549" href="#0000796" shape="RECT"/>
<area alt="" coords="5,197,133,245" href="#0032991" shape="RECT"/>
<area alt="" coords="155,5,37,29" href="#0005575" shape="RECT"/>
<area alt="" coords="116,101,193,149" href="#0044422" shape="RECT"/>
<area alt="" coords="217,101,289,149" href="#0044464" shape="RECT"/>
<area alt="" coords="135,405,239,453" href="#0044427" shape="RECT"/>
</map>
<p align="center"><img alt="" border="0" src="/static/analysis/data1/go/50/GOCOMPONENT.png" usemap="#goPathImage"/></p>
<p>Â </p>
<table border="1" id="table1" width="100%">
<tr>
<td align="center"><b>GO term</b></td>
<td align="center"><b>Description</b></td>
<td align="center" width="120"><b><a href="#p_value_info">P-value</a></b></td>
<td align="center" width="120"><b><a href="#fdr_info">FDR q-value</a></b></td>
<td align="center" width="230"><b><a href="#enrich_info"> Enrichment (N, B, n, b) </a></b> </td>
<td align="center"><b><a href="#genes_info"> Genes </a></b></td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0000796&amp;view=details" target="_blank">GO:0000796</a><a name="0000796"></a></td>
<td>condensin complex</td>
<td align="center">3.88E-4</td>
<td align="center">6.68E-1</td>
<td align="center">9.86 (8795,8,446,4)</td>
<td>
<a href="javascript:toggle('elements_GO:0000796')" id="ctrl_elements_GO:0000796">[+] Show genes</a><br/>
<div id="elements_GO:0000796" style="display:none"> <font size="1">

NCAPH2  -  non-smc condensin ii complex, subunit h2<br/>

NCAPG  -  non-smc condensin i complex, subunit g<br/>

SMC2  -  structural maintenance of chromosomes 2<br/>

NCAPD2  -  non-smc condensin i complex, subunit d2<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0044427&amp;view=details" target="_blank">GO:0044427</a><a name="0044427"></a></td>
<td>chromosomal part</td>
<td align="center">8.71E-4</td>
<td align="center">7.49E-1</td>
<td align="center">1.46 (8795,906,446,67)</td>
<td>
<a href="javascript:toggle('elements_GO:0044427')" id="ctrl_elements_GO:0044427">[+] Show genes</a><br/>
<div id="elements_GO:0044427" style="display:none"> <font size="1">

DHX9  -  deah (asp-glu-ala-his) box helicase 9<br/>

DONSON  -  downstream neighbor of son<br/>

FAM111B  -  family with sequence similarity 111, member b<br/>

FMR1  -  fragile x mental retardation 1<br/>

RFC5  -  replication factor c (activator 1) 5, 36.5kda<br/>

RFC1  -  replication factor c (activator 1) 1, 145kda<br/>

ORC1  -  origin recognition complex, subunit 1<br/>

SGOL1  -  shugoshin-like 1 (s. pombe)<br/>

BRCA1  -  breast cancer 1, early onset<br/>

CDT1  -  chromatin licensing and dna replication factor 1<br/>

SPAG5  -  sperm associated antigen 5<br/>

RELA  -  v-rel avian reticuloendotheliosis viral oncogene homolog a<br/>

HIST2H2AC  -  histone cluster 2, h2ac<br/>

MLH1  -  mutl homolog 1, colon cancer, nonpolyposis type 2 (e. coli)<br/>

SMC2  -  structural maintenance of chromosomes 2<br/>

RMI1  -  recq mediated genome instability 1<br/>

ZWINT  -  zw10 interacting kinetochore protein<br/>

FOXM1  -  forkhead box m1<br/>

NASP  -  nuclear autoantigenic sperm protein (histone-binding)<br/>

RPA3  -  replication protein a3, 14kda<br/>

MSH6  -  muts homolog 6 (e. coli)<br/>

SPC24  -  spc24, ndc80 kinetochore complex component<br/>

NCAPG  -  non-smc condensin i complex, subunit g<br/>

NCAPH2  -  non-smc condensin ii complex, subunit h2<br/>

NFYB  -  nuclear transcription factor y, beta<br/>

CHD4  -  chromodomain helicase dna binding protein 4<br/>

RAD18  -  rad18 homolog (s. cerevisiae)<br/>

CHD3  -  chromodomain helicase dna binding protein 3<br/>

NUP43  -  nucleoporin 43kda<br/>

WHSC1  -  wolf-hirschhorn syndrome candidate 1<br/>

IFFO1  -  intermediate filament family orphan 1<br/>

RAD51  -  rad51 recombinase<br/>

FOXP4  -  forkhead box p4<br/>

RAD21  -  rad21 homolog (s. pombe)<br/>

EZH2  -  enhancer of zeste homolog 2 (drosophila)<br/>

DDX23  -  dead (asp-glu-ala-asp) box polypeptide 23<br/>

EXOSC3  -  exosome component 3<br/>

RAD51AP1  -  rad51 associated protein 1<br/>

CENPP  -  centromere protein p<br/>

HIST1H1B  -  histone cluster 1, h1b<br/>

NCAPD2  -  non-smc condensin i complex, subunit d2<br/>

MIS12  -  mis12 kinetochore complex component<br/>

CTR9  -  ctr9, paf1/rna polymerase ii complex component<br/>

ESCO2  -  establishment of sister chromatid cohesion n-acetyltransferase 2<br/>

ARID1A  -  at rich interactive domain 1a (swi-like)<br/>

E2F8  -  e2f transcription factor 8<br/>

NUP98  -  nucleoporin 98kda<br/>

MCM10  -  minichromosome maintenance complex component 10<br/>

DEAF1  -  deaf1 transcription factor<br/>

PHF12  -  phd finger protein 12<br/>

RNF2  -  ring finger protein 2<br/>

RNF8  -  ring finger protein 8, e3 ubiquitin protein ligase<br/>

PCNA  -  proliferating cell nuclear antigen<br/>

BUB1B  -  bub1 mitotic checkpoint serine/threonine kinase b<br/>

SMAD2  -  smad family member 2<br/>

CHTF18  -  ctf18, chromosome transmission fidelity factor 18 homolog (s. cerevisiae)<br/>

GINS1  -  gins complex subunit 1 (psf1 homolog)<br/>

HIST1H2BK  -  histone cluster 1, h2bk<br/>

HIST1H2AH  -  histone cluster 1, h2ah<br/>

E2F2  -  e2f transcription factor 2<br/>

THOC1  -  tho complex 1<br/>

HELLS  -  helicase, lymphoid-specific<br/>

WRAP53  -  wd repeat containing, antisense to tp53<br/>

TCF3  -  transcription factor 3<br/>

CSNK2A1  -  casein kinase 2, alpha 1 polypeptide<br/>

KIF2C  -  kinesin family member 2c<br/>

YY1  -  yy1 transcription factor<br/>
</font></div>
</td>
</tr>
</table>
<p>Species used: Homo sapiens</p><p>The system has recognized 9488 genes out of 9956 gene terms entered by the user.<br/> 9488 genes were recognized by gene symbol and 0 genes by other gene IDs .<br/>460 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 9028 genes.<br/>Only 8795 of these genes are associated with a GO term.<br/></p>
<br/><a href="GOCOMPONENT.xls"> Output in Microsoft Excel format</a><br/>
<p> The GOrilla database is periodically updated using the <a href="http://www.geneontology.org/">GO database</a> and other sources.<br/>

The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 

Jan 16, 2023

). You can bookmark this page and come back to it later.

</p>
<a name="p_value_info"></a><br/><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 1721 GO terms.<br/>
<a name="fdr_info"></a><br/><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br/>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br/>
<a name="enrich_info"></a><br/><b>Enrichment (N, B, n, b)</b> is defined as follows:<br/>N - is the total number of genes<br/>B - is the total number of genes associated with a specific GO term<br/>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br/>b - is the number of genes in the intersection<br/>Enrichment = (b/n) / (B/N)<br/>
<a name="genes_info"></a><br/><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br/>Each gene name is specified by gene symbol followed by a short description of the gene <br/>
<p>
<a href="http://cbl-gorilla.cs.technion.ac.il/"> Back to the GOrilla main page </a>
</p>
</body>
</html>
