<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>
<title>

Results

</title>
<meta content="text/html;charset=utf-8" http-equiv="Content-Type"/>
<script type="text/javascript">

function toggle(item) {

obj=document.getElementById(item);

visible=(obj.style.display!="none")

key=document.getElementById("ctrl_" + item);

if (visible) {

    obj.style.display="none";

key.innerHTML="[+] Show genes";

}

else {

obj.style.display="block";

key.innerHTML="[-] Hide genes";

}

}

</script>
</head>
<body>
<p><center><b>
</b></center></p>
<table align="center" border="1" id="tab:pval_legend" width="80%">
<caption><em>P-value color scale</em></caption>
<tr>
<td align="center" width="140"> &gt; 10<sup>-3</sup></td>
<td align="center" bgcolor="#FFFFCC" width="140"> 10<sup>-3</sup> to 10<sup>-5</sup></td>
<td align="center" bgcolor="#FFCC66" width="140"> 10<sup>-5</sup> to 10<sup>-7</sup></td>
<td align="center" bgcolor="#FF9900" width="140"> 10<sup>-7</sup> to 10<sup>-9</sup></td>
<td align="center" bgcolor="#FF3300" width="140"> &lt; 10<sup>-9</sup></td>
</tr>
</table>
<map name="goPathImage">
<area alt="" coords="29,441,120,538" href="#0004553" shape="RECT"/>
<area alt="" coords="24,587,125,635" href="#0015923" shape="RECT"/>
<area alt="" coords="5,683,144,731" href="#0004559" shape="RECT"/>
<area alt="" coords="140,319,247,367" href="#0051787" shape="RECT"/>
<area alt="" coords="33,294,116,391" href="#0016798" shape="RECT"/>
<area alt="" coords="33,197,116,245" href="#0016787" shape="RECT"/>
<area alt="" coords="271,319,377,367" href="#0051082" shape="RECT"/>
<area alt="" coords="140,197,247,245" href="#0005515" shape="RECT"/>
<area alt="" coords="48,101,123,149" href="#0003824" shape="RECT"/>
<area alt="" coords="152,101,224,149" href="#0005488" shape="RECT"/>
<area alt="" coords="136,5,22,29" href="#0003674" shape="RECT"/>
</map>
<p align="center"><img alt="" border="0" src="/static/analysis/data1/go/17/GOFUNCTION.png" usemap="#goPathImage"/></p>
<p>Â </p>
<table border="1" id="table1" width="100%">
<tr>
<td align="center"><b>GO term</b></td>
<td align="center"><b>Description</b></td>
<td align="center" width="120"><b><a href="#p_value_info">P-value</a></b></td>
<td align="center" width="120"><b><a href="#fdr_info">FDR q-value</a></b></td>
<td align="center" width="230"><b><a href="#enrich_info"> Enrichment (N, B, n, b) </a></b> </td>
<td align="center"><b><a href="#genes_info"> Genes </a></b></td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0051787&amp;view=details" target="_blank">GO:0051787</a><a name="0051787"></a></td>
<td>misfolded protein binding</td>
<td align="center">5.96E-6</td>
<td align="center">2.19E-2</td>
<td align="center">18.38 (8795,23,104,5)</td>
<td>
<a href="javascript:toggle('elements_GO:0051787')" id="ctrl_elements_GO:0051787">[+] Show genes</a><br/>
<div id="elements_GO:0051787" style="display:none"> <font size="1">

EDEM1  -  er degradation enhancer, mannosidase alpha-like 1<br/>

DNAJC3  -  dnaj (hsp40) homolog, subfamily c, member 3<br/>

HSPA5  -  heat shock 70kda protein 5 (glucose-regulated protein, 78kda)<br/>

DERL2  -  derlin 2<br/>

DNAJB9  -  dnaj (hsp40) homolog, subfamily b, member 9<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0004559&amp;view=details" target="_blank">GO:0004559</a><a name="0004559"></a></td>
<td>alpha-mannosidase activity</td>
<td align="center">3.27E-4</td>
<td align="center">6E-1</td>
<td align="center">21.14 (8795,12,104,3)</td>
<td>
<a href="javascript:toggle('elements_GO:0004559')" id="ctrl_elements_GO:0004559">[+] Show genes</a><br/>
<div id="elements_GO:0004559" style="display:none"> <font size="1">

MANEA  -  mannosidase, endo-alpha<br/>

EDEM1  -  er degradation enhancer, mannosidase alpha-like 1<br/>

EDEM3  -  er degradation enhancer, mannosidase alpha-like 3<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0015923&amp;view=details" target="_blank">GO:0015923</a><a name="0015923"></a></td>
<td>mannosidase activity</td>
<td align="center">4.21E-4</td>
<td align="center">5.16E-1</td>
<td align="center">19.52 (8795,13,104,3)</td>
<td>
<a href="javascript:toggle('elements_GO:0015923')" id="ctrl_elements_GO:0015923">[+] Show genes</a><br/>
<div id="elements_GO:0015923" style="display:none"> <font size="1">

MANEA  -  mannosidase, endo-alpha<br/>

EDEM1  -  er degradation enhancer, mannosidase alpha-like 1<br/>

EDEM3  -  er degradation enhancer, mannosidase alpha-like 3<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0051082&amp;view=details" target="_blank">GO:0051082</a><a name="0051082"></a></td>
<td>unfolded protein binding</td>
<td align="center">4.86E-4</td>
<td align="center">4.46E-1</td>
<td align="center">5.97 (8795,85,104,6)</td>
<td>
<a href="javascript:toggle('elements_GO:0051082')" id="ctrl_elements_GO:0051082">[+] Show genes</a><br/>
<div id="elements_GO:0051082" style="display:none"> <font size="1">

HSP90B1  -  heat shock protein 90kda beta (grp94), member 1<br/>

ERLEC1  -  endoplasmic reticulum lectin 1<br/>

PPIB  -  peptidylprolyl isomerase b (cyclophilin b)<br/>

HSPA5  -  heat shock 70kda protein 5 (glucose-regulated protein, 78kda)<br/>

SIL1  -  sil1 nucleotide exchange factor<br/>

LMAN1  -  lectin, mannose-binding, 1<br/>
</font></div>
</td>
</tr>
</table>
<p>Species used: Homo sapiens</p><p>The system has recognized 9136 genes out of 9583 gene terms entered by the user.<br/> 9136 genes were recognized by gene symbol and 0 genes by other gene IDs .<br/>108 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 9028 genes.<br/>Only 8795 of these genes are associated with a GO term.<br/></p>
<br/><a href="GOFUNCTION.xls"> Output in Microsoft Excel format</a><br/>
<p> The GOrilla database is periodically updated using the <a href="http://www.geneontology.org/">GO database</a> and other sources.<br/>

The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 

Jan 16, 2023

). You can bookmark this page and come back to it later.

</p>
<a name="p_value_info"></a><br/><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 3672 GO terms.<br/>
<a name="fdr_info"></a><br/><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br/>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br/>
<a name="enrich_info"></a><br/><b>Enrichment (N, B, n, b)</b> is defined as follows:<br/>N - is the total number of genes<br/>B - is the total number of genes associated with a specific GO term<br/>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br/>b - is the number of genes in the intersection<br/>Enrichment = (b/n) / (B/N)<br/>
<a name="genes_info"></a><br/><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br/>Each gene name is specified by gene symbol followed by a short description of the gene <br/>
<p>
<a href="http://cbl-gorilla.cs.technion.ac.il/"> Back to the GOrilla main page </a>
</p>
</body>
</html>
