<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>
<title>

Results

</title>
<meta content="text/html;charset=utf-8" http-equiv="Content-Type"/>
<script type="text/javascript">

function toggle(item) {

obj=document.getElementById(item);

visible=(obj.style.display!="none")

key=document.getElementById("ctrl_" + item);

if (visible) {

    obj.style.display="none";

key.innerHTML="[+] Show genes";

}

else {

obj.style.display="block";

key.innerHTML="[-] Hide genes";

}

}

</script>
</head>
<body>
<p><center><b>
</b></center></p>
<table align="center" border="1" id="tab:pval_legend" width="80%">
<caption><em>P-value color scale</em></caption>
<tr>
<td align="center" width="140"> &gt; 10<sup>-3</sup></td>
<td align="center" bgcolor="#FFFFCC" width="140"> 10<sup>-3</sup> to 10<sup>-5</sup></td>
<td align="center" bgcolor="#FFCC66" width="140"> 10<sup>-5</sup> to 10<sup>-7</sup></td>
<td align="center" bgcolor="#FF9900" width="140"> 10<sup>-7</sup> to 10<sup>-9</sup></td>
<td align="center" bgcolor="#FF3300" width="140"> &lt; 10<sup>-9</sup></td>
</tr>
</table>
<map name="goPathImage">
<area alt="" coords="13,198,96,261" href="#0098772" shape="RECT"/>
<area alt="" coords="52,502,177,565" href="#0005085" shape="RECT"/>
<area alt="" coords="125,101,197,149" href="#0005488" shape="RECT"/>
<area alt="" coords="120,205,227,253" href="#0005515" shape="RECT"/>
<area alt="" coords="113,309,225,357" href="#0019899" shape="RECT"/>
<area alt="" coords="107,405,221,453" href="#0051020" shape="RECT"/>
<area alt="" coords="119,5,5,29" href="#0003674" shape="RECT"/>
</map>
<p align="center"><img alt="" border="0" src="/static/analysis/data1/go/44/GOFUNCTION.png" usemap="#goPathImage"/></p>
<p>Â </p>
<table border="1" id="table1" width="100%">
<tr>
<td align="center"><b>GO term</b></td>
<td align="center"><b>Description</b></td>
<td align="center" width="120"><b><a href="#p_value_info">P-value</a></b></td>
<td align="center" width="120"><b><a href="#fdr_info">FDR q-value</a></b></td>
<td align="center" width="230"><b><a href="#enrich_info"> Enrichment (N, B, n, b) </a></b> </td>
<td align="center"><b><a href="#genes_info"> Genes </a></b></td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0051020&amp;view=details" target="_blank">GO:0051020</a><a name="0051020"></a></td>
<td>GTPase binding</td>
<td align="center">2.41E-5</td>
<td align="center">8.87E-2</td>
<td align="center">1.99 (8795,259,666,39)</td>
<td>
<a href="javascript:toggle('elements_GO:0051020')" id="ctrl_elements_GO:0051020">[+] Show genes</a><br/>
<div id="elements_GO:0051020" style="display:none"> <font size="1">

DNMBP  -  dynamin binding protein<br/>

PICALM  -  phosphatidylinositol binding clathrin assembly protein<br/>

SERGEF  -  secretion regulating guanine nucleotide exchange factor<br/>

DENND4C  -  denn/madd domain containing 4c<br/>

DENND5A  -  denn/madd domain containing 5a<br/>

DIS3  -  dis3 mitotic control homolog (s. cerevisiae)<br/>

USP6NL  -  usp6 n-terminal like<br/>

PEX5  -  peroxisomal biogenesis factor 5<br/>

RGL1  -  ral guanine nucleotide dissociation stimulator-like 1<br/>

CYTH2  -  cytohesin 2<br/>

LAMTOR4  -  late endosomal/lysosomal adaptor, mapk and mtor activator 4<br/>

TMEM127  -  transmembrane protein 127<br/>

EIF2B3  -  eukaryotic translation initiation factor 2b, subunit 3 gamma, 58kda<br/>

KIF3B  -  kinesin family member 3b<br/>

ARHGAP1  -  rho gtpase activating protein 1<br/>

IPO8  -  importin 8<br/>

RAB11FIP4  -  rab11 family interacting protein 4 (class ii)<br/>

SH3BP5  -  sh3-domain binding protein 5 (btk-associated)<br/>

PREX1  -  phosphatidylinositol-3,4,5-trisphosphate-dependent rac exchange factor 1<br/>

ATG14  -  autophagy related 14<br/>

DAPK3  -  death-associated protein kinase 3<br/>

WBSCR16  -  williams-beuren syndrome chromosome region 16<br/>

RIC8B  -  ric8 guanine nucleotide exchange factor b<br/>

RAB34  -  rab34, member ras oncogene family<br/>

RGL2  -  ral guanine nucleotide dissociation stimulator-like 2<br/>

PLCE1  -  phospholipase c, epsilon 1<br/>

WDR41  -  wd repeat domain 41<br/>

GGA1  -  golgi-associated, gamma adaptin ear containing, arf binding protein 1<br/>

ARHGEF2  -  rho/rac guanine nucleotide exchange factor (gef) 2<br/>

SPATA13  -  spermatogenesis associated 13<br/>

ARHGEF18  -  rho/rac guanine nucleotide exchange factor (gef) 18<br/>

TRIOBP  -  trio and f-actin binding protein<br/>

RINL  -  ras and rab interactor-like<br/>

DNM1L  -  dynamin 1-like<br/>

VAV2  -  vav 2 guanine nucleotide exchange factor<br/>

KCTD13  -  potassium channel tetramerization domain containing 13<br/>

RAPGEF6  -  rap guanine nucleotide exchange factor (gef) 6<br/>

RABIF  -  rab interacting factor<br/>

AMBRA1  -  autophagy/beclin-1 regulator 1<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0005085&amp;view=details" target="_blank">GO:0005085</a><a name="0005085"></a></td>
<td>guanyl-nucleotide exchange factor activity</td>
<td align="center">4.71E-5</td>
<td align="center">8.65E-2</td>
<td align="center">2.45 (8795,124,666,23)</td>
<td>
<a href="javascript:toggle('elements_GO:0005085')" id="ctrl_elements_GO:0005085">[+] Show genes</a><br/>
<div id="elements_GO:0005085" style="display:none"> <font size="1">

SH3BP5  -  sh3-domain binding protein 5 (btk-associated)<br/>

PREX1  -  phosphatidylinositol-3,4,5-trisphosphate-dependent rac exchange factor 1<br/>

WBSCR16  -  williams-beuren syndrome chromosome region 16<br/>

DNMBP  -  dynamin binding protein<br/>

RIC8B  -  ric8 guanine nucleotide exchange factor b<br/>

RGL2  -  ral guanine nucleotide dissociation stimulator-like 2<br/>

PLCE1  -  phospholipase c, epsilon 1<br/>

SERGEF  -  secretion regulating guanine nucleotide exchange factor<br/>

WDR41  -  wd repeat domain 41<br/>

DENND4C  -  denn/madd domain containing 4c<br/>

DENND5A  -  denn/madd domain containing 5a<br/>

ARHGEF2  -  rho/rac guanine nucleotide exchange factor (gef) 2<br/>

DIS3  -  dis3 mitotic control homolog (s. cerevisiae)<br/>

SPATA13  -  spermatogenesis associated 13<br/>

ARHGEF18  -  rho/rac guanine nucleotide exchange factor (gef) 18<br/>

RINL  -  ras and rab interactor-like<br/>

RGL1  -  ral guanine nucleotide dissociation stimulator-like 1<br/>

VAV2  -  vav 2 guanine nucleotide exchange factor<br/>

CYTH2  -  cytohesin 2<br/>

LAMTOR4  -  late endosomal/lysosomal adaptor, mapk and mtor activator 4<br/>

RAPGEF6  -  rap guanine nucleotide exchange factor (gef) 6<br/>

EIF2B3  -  eukaryotic translation initiation factor 2b, subunit 3 gamma, 58kda<br/>

RABIF  -  rab interacting factor<br/>
</font></div>
</td>
</tr>
</table>
<p>Species used: Homo sapiens</p><p>The system has recognized 9715 genes out of 10194 gene terms entered by the user.<br/> 9715 genes were recognized by gene symbol and 0 genes by other gene IDs .<br/>687 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 9028 genes.<br/>Only 8795 of these genes are associated with a GO term.<br/></p>
<br/><a href="GOFUNCTION.xls"> Output in Microsoft Excel format</a><br/>
<p> The GOrilla database is periodically updated using the <a href="http://www.geneontology.org/">GO database</a> and other sources.<br/>

The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 

Jan 16, 2023

). You can bookmark this page and come back to it later.

</p>
<a name="p_value_info"></a><br/><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 3672 GO terms.<br/>
<a name="fdr_info"></a><br/><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br/>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br/>
<a name="enrich_info"></a><br/><b>Enrichment (N, B, n, b)</b> is defined as follows:<br/>N - is the total number of genes<br/>B - is the total number of genes associated with a specific GO term<br/>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br/>b - is the number of genes in the intersection<br/>Enrichment = (b/n) / (B/N)<br/>
<a name="genes_info"></a><br/><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br/>Each gene name is specified by gene symbol followed by a short description of the gene <br/>
<p>
<a href="http://cbl-gorilla.cs.technion.ac.il/"> Back to the GOrilla main page </a>
</p>
</body>
</html>
