<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>
<title>

Results

</title>
<meta content="text/html;charset=utf-8" http-equiv="Content-Type"/>
<script type="text/javascript">

function toggle(item) {

obj=document.getElementById(item);

visible=(obj.style.display!="none")

key=document.getElementById("ctrl_" + item);

if (visible) {

    obj.style.display="none";

key.innerHTML="[+] Show genes";

}

else {

obj.style.display="block";

key.innerHTML="[-] Hide genes";

}

}

</script>
</head>
<body>
<p><center><b>
</b></center></p>
<table align="center" border="1" id="tab:pval_legend" width="80%">
<caption><em>P-value color scale</em></caption>
<tr>
<td align="center" width="140"> &gt; 10<sup>-3</sup></td>
<td align="center" bgcolor="#FFFFCC" width="140"> 10<sup>-3</sup> to 10<sup>-5</sup></td>
<td align="center" bgcolor="#FFCC66" width="140"> 10<sup>-5</sup> to 10<sup>-7</sup></td>
<td align="center" bgcolor="#FF9900" width="140"> 10<sup>-7</sup> to 10<sup>-9</sup></td>
<td align="center" bgcolor="#FF3300" width="140"> &lt; 10<sup>-9</sup></td>
</tr>
</table>
<map name="goPathImage">
<area alt="" coords="209,646,311,709" href="#0031749" shape="RECT"/>
<area alt="" coords="72,317,184,365" href="#0019899" shape="RECT"/>
<area alt="" coords="57,429,172,477" href="#0051020" shape="RECT"/>
<area alt="" coords="5,534,131,597" href="#0005085" shape="RECT"/>
<area alt="" coords="57,213,183,261" href="#0016247" shape="RECT"/>
<area alt="" coords="79,102,161,165" href="#0098772" shape="RECT"/>
<area alt="" coords="196,422,324,485" href="#0001664" shape="RECT"/>
<area alt="" coords="219,534,301,597" href="#0050780" shape="RECT"/>
<area alt="" coords="221,310,299,373" href="#0005102" shape="RECT"/>
<area alt="" coords="204,109,276,157" href="#0005488" shape="RECT"/>
<area alt="" coords="207,213,313,261" href="#0005515" shape="RECT"/>
<area alt="" coords="180,5,66,29" href="#0003674" shape="RECT"/>
</map>
<p align="center"><img alt="" border="0" src="/static/analysis/data2/go/6/GOFUNCTION.png" usemap="#goPathImage"/></p>
<p>Â </p>
<table border="1" id="table1" width="100%">
<tr>
<td align="center"><b>GO term</b></td>
<td align="center"><b>Description</b></td>
<td align="center" width="120"><b><a href="#p_value_info">P-value</a></b></td>
<td align="center" width="120"><b><a href="#fdr_info">FDR q-value</a></b></td>
<td align="center" width="230"><b><a href="#enrich_info"> Enrichment (N, B, n, b) </a></b> </td>
<td align="center"><b><a href="#genes_info"> Genes </a></b></td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0050780&amp;view=details" target="_blank">GO:0050780</a><a name="0050780"></a></td>
<td>dopamine receptor binding</td>
<td align="center">6.71E-5</td>
<td align="center">3.02E-1</td>
<td align="center">37.53 (16689,23,58,3)</td>
<td>
<a href="javascript:toggle('elements_GO:0050780')" id="ctrl_elements_GO:0050780">[+] Show genes</a><br/>
<div id="elements_GO:0050780" style="display:none"> <font size="1">

Grin2b  -  glutamate receptor, ionotropic, nmda2b (epsilon 2)<br/>

Ppp1r9b  -  protein phosphatase 1, regulatory subunit 9b<br/>

Palm  -  paralemmin<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0016247&amp;view=details" target="_blank">GO:0016247</a><a name="0016247"></a></td>
<td>channel regulator activity</td>
<td align="center">1.04E-4</td>
<td align="center">2.35E-1</td>
<td align="center">10.66 (16689,135,58,5)</td>
<td>
<a href="javascript:toggle('elements_GO:0016247')" id="ctrl_elements_GO:0016247">[+] Show genes</a><br/>
<div id="elements_GO:0016247" style="display:none"> <font size="1">

Hpcal4  -  hippocalcin-like 4<br/>

Fgf13  -  fibroblast growth factor 13<br/>

Cacnb3  -  calcium channel, voltage-dependent, beta 3 subunit<br/>

Kcnip2  -  kv channel-interacting protein 2<br/>

Cnih2  -  cornichon homolog 2 (drosophila)<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0031749&amp;view=details" target="_blank">GO:0031749</a><a name="0031749"></a></td>
<td>D2 dopamine receptor binding</td>
<td align="center">1.76E-4</td>
<td align="center">2.64E-1</td>
<td align="center">95.91 (16689,6,58,2)</td>
<td>
<a href="javascript:toggle('elements_GO:0031749')" id="ctrl_elements_GO:0031749">[+] Show genes</a><br/>
<div id="elements_GO:0031749" style="display:none"> <font size="1">

Grin2b  -  glutamate receptor, ionotropic, nmda2b (epsilon 2)<br/>

Ppp1r9b  -  protein phosphatase 1, regulatory subunit 9b<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0005085&amp;view=details" target="_blank">GO:0005085</a><a name="0005085"></a></td>
<td>guanyl-nucleotide exchange factor activity</td>
<td align="center">5.61E-4</td>
<td align="center">6.3E-1</td>
<td align="center">7.42 (16689,194,58,5)</td>
<td>
<a href="javascript:toggle('elements_GO:0005085')" id="ctrl_elements_GO:0005085">[+] Show genes</a><br/>
<div id="elements_GO:0005085" style="display:none"> <font size="1">

Kalrn  -  kalirin, rhogef kinase<br/>

Rasgef1a  -  rasgef domain family, member 1a<br/>

Rapgefl1  -  rap guanine nucleotide exchange factor (gef)-like 1<br/>

Iqsec2  -  iq motif and sec7 domain 2<br/>

Psd  -  pleckstrin and sec7 domain containing<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0098772&amp;view=details" target="_blank">GO:0098772</a><a name="0098772"></a></td>
<td>molecular function regulator</td>
<td align="center">5.67E-4</td>
<td align="center">5.09E-1</td>
<td align="center">2.65 (16689,1522,58,14)</td>
<td>
<a href="javascript:toggle('elements_GO:0098772')" id="ctrl_elements_GO:0098772">[+] Show genes</a><br/>
<div id="elements_GO:0098772" style="display:none"> <font size="1">

Agap2  -  arfgap with gtpase domain, ankyrin repeat and ph domain 2<br/>

Sipa1l1  -  signal-induced proliferation-associated 1 like 1<br/>

Psd  -  pleckstrin and sec7 domain containing<br/>

Cnih2  -  cornichon homolog 2 (drosophila)<br/>

Rin1  -  ras and rab interactor 1<br/>

Hpcal4  -  hippocalcin-like 4<br/>

Fgf13  -  fibroblast growth factor 13<br/>

Kalrn  -  kalirin, rhogef kinase<br/>

Rasgef1a  -  rasgef domain family, member 1a<br/>

Cacnb3  -  calcium channel, voltage-dependent, beta 3 subunit<br/>

Kcnip2  -  kv channel-interacting protein 2<br/>

Rapgefl1  -  rap guanine nucleotide exchange factor (gef)-like 1<br/>

Iqsec2  -  iq motif and sec7 domain 2<br/>

Ppp1r9b  -  protein phosphatase 1, regulatory subunit 9b<br/>
</font></div>
</td>
</tr>
</table>
<p>Species used: Mus musculus</p><p>The system has recognized 18003 genes out of 21009 gene terms entered by the user.<br/> 18003 genes were recognized by gene symbol and 0 genes by other gene IDs .<br/>76 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 17927 genes.<br/>Only 16689 of these genes are associated with a GO term.<br/></p>
<br/><a href="GOFUNCTION.xls"> Output in Microsoft Excel format</a><br/>
<p> The GOrilla database is periodically updated using the <a href="http://www.geneontology.org/">GO database</a> and other sources.<br/>

The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 

Jan 16, 2023

). You can bookmark this page and come back to it later.

</p>
<a name="p_value_info"></a><br/><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 4496 GO terms.<br/>
<a name="fdr_info"></a><br/><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br/>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br/>
<a name="enrich_info"></a><br/><b>Enrichment (N, B, n, b)</b> is defined as follows:<br/>N - is the total number of genes<br/>B - is the total number of genes associated with a specific GO term<br/>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br/>b - is the number of genes in the intersection<br/>Enrichment = (b/n) / (B/N)<br/>
<a name="genes_info"></a><br/><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br/>Each gene name is specified by gene symbol followed by a short description of the gene <br/>
<p>
<a href="http://cbl-gorilla.cs.technion.ac.il/"> Back to the GOrilla main page </a>
</p>
</body>
</html>
