<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
<title>
Results
</title>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" >
<script type='text/javascript'>
function toggle(item) {
obj=document.getElementById(item);
visible=(obj.style.display!="none")
key=document.getElementById("ctrl_" + item);
if (visible) {
    obj.style.display="none";
key.innerHTML="[+] Show genes";
}
else {
obj.style.display="block";
key.innerHTML="[-] Hide genes";
}
}
</script>
</head>
<body>
<p><center><b>

</b></center></p>
<table border="1" width="80%" id="tab:pval_legend" align=center>
<CAPTION><EM>P-value color scale</EM></CAPTION>
<tr>
	<td  width=140 align=center> > 10<sup>-3</sup></td>
	<td  width=140 align=center bgcolor='#FFFFCC'> 10<sup>-3</sup> to 10<sup>-5</sup></td>
	<td  width=140 align=center bgcolor='#FFCC66'> 10<sup>-5</sup> to 10<sup>-7</sup></td>
	<td  width=140 align=center bgcolor='#FF9900'> 10<sup>-7</sup> to 10<sup>-9</sup></td>
	<td  width=140 align=center bgcolor='#FF3300'> < 10<sup>-9</sup></td>
</tr >
</table>
<MAP NAME='goPathImage'>
<AREA SHAPE='RECT' COORDS='75,309,168,357' HREF='#0003676' ALT=''>
<AREA SHAPE='RECT' COORDS='75,405,168,453' HREF='#0003723' ALT=''>
<AREA SHAPE='RECT' COORDS='5,198,112,261' HREF='#0097159' ALT=''>
<AREA SHAPE='RECT' COORDS='136,198,232,261' HREF='#1901363' ALT=''>
<AREA SHAPE='RECT' COORDS='85,101,157,149' HREF='#0005488' ALT=''>
<AREA SHAPE='RECT' COORDS='121,5,8,29' HREF='#0003674' ALT=''>
</MAP>
<p align="center"><img border="0" src="GOFUNCTION.png" usemap='#goPathImage' ALT=''></p>
<p>&nbsp;</p>
<table border='1' width='100%' id='table1'>
	<tr>
		<td Align=center ><b>GO term</b></td>
		<td Align=center><b>Description</b></td>
		<td width=120 Align=center><b><a href='#p_value_info'>P-value</a></b></td> 
		<td width=120 Align=center><b><a href='#fdr_info'>FDR q-value</a></b></td> 
		<td width=230 Align=center><b><a href='#enrich_info'> Enrichment (N, B, n, b) </a></b> </td>
		<td Align=center><b><a href='#genes_info'> Genes </a></b></td>
	</tr>
<tr>
<td><a href='http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0003723&amp;view=details' target="_blank">GO:0003723</a><a name=0003723></a></td>
<td>RNA binding</td>
<td  align=center>2.12E-4</td>
<td  align=center>9.56E-1</td>
<td  align=center>2.07 (15353,1325,151,27)</td>
<td>
<a id="ctrl_elements_GO:0003723" href="javascript:toggle('elements_GO:0003723')">[+] Show genes</a><br>
<div id="elements_GO:0003723" style="display:none"> <font size=1>
LAS1L  -  las1-like (s. cerevisiae)<br>
MKRN1  -  makorin ring finger protein 1<br>
ACAA2  -  acetyl-coa acyltransferase 2<br>
FXR2  -  fragile x mental retardation, autosomal homolog 2<br>
CCNT1  -  cyclin t1<br>
BRIX1  -  brx1, biogenesis of ribosomes, homolog (s. cerevisiae)<br>
KTN1  -  kinectin 1 (kinesin receptor)<br>
ADARB1  -  adenosine deaminase, rna-specific, b1<br>
SAMD4A  -  sterile alpha motif domain containing 4a<br>
MAP4  -  microtubule-associated protein 4<br>
MEX3B  -  mex-3 rna binding family member b<br>
KIF1C  -  kinesin family member 1c<br>
CKAP4  -  cytoskeleton-associated protein 4<br>
RBM15B  -  rna binding motif protein 15b<br>
EIF3G  -  eukaryotic translation initiation factor 3, subunit g<br>
ESF1  -  esf1, nucleolar pre-rrna processing protein, homolog (s. cerevisiae)<br>
THOC7  -  tho complex 7 homolog (drosophila)<br>
ANGEL2  -  angel homolog 2 (drosophila)<br>
PABPC4  -  poly(a) binding protein, cytoplasmic 4 (inducible form)<br>
SRPK1  -  srsf protein kinase 1<br>
GFM1  -  g elongation factor, mitochondrial 1<br>
ROCK2  -  rho-associated, coiled-coil containing protein kinase 2<br>
ALYREF  -  aly/ref export factor<br>
PCBP4  -  poly(rc) binding protein 4<br>
EIF5B  -  eukaryotic translation initiation factor 5b<br>
ZNF74  -  zinc finger protein 74<br>
SF3B3  -  splicing factor 3b, subunit 3, 130kda<br>
</FONT></div>
</td>
</tr>
<tr>
<td><a href='http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0003676&amp;view=details' target="_blank">GO:0003676</a><a name=0003676></a></td>
<td>nucleic acid binding</td>
<td  align=center>7.63E-4</td>
<td  align=center>1E0</td>
<td  align=center>1.53 (15353,3246,151,49)</td>
<td>
<a id="ctrl_elements_GO:0003676" href="javascript:toggle('elements_GO:0003676')">[+] Show genes</a><br>
<div id="elements_GO:0003676" style="display:none"> <font size=1>
SP110  -  sp110 nuclear body protein<br>
ZNF605  -  zinc finger protein 605<br>
MKRN1  -  makorin ring finger protein 1<br>
ACAA2  -  acetyl-coa acyltransferase 2<br>
REV3L  -  rev3-like, polymerase (dna directed), zeta, catalytic subunit<br>
AKAP9  -  a kinase (prka) anchor protein (yotiao) 9<br>
BRIX1  -  brx1, biogenesis of ribosomes, homolog (s. cerevisiae)<br>
MAP4  -  microtubule-associated protein 4<br>
MEX3B  -  mex-3 rna binding family member b<br>
KIF1C  -  kinesin family member 1c<br>
CKAP4  -  cytoskeleton-associated protein 4<br>
ZFPM2  -  zinc finger protein, fog family member 2<br>
EIF3G  -  eukaryotic translation initiation factor 3, subunit g<br>
THOC7  -  tho complex 7 homolog (drosophila)<br>
ETV6  -  ets variant 6<br>
NFYC  -  nuclear transcription factor y, gamma<br>
CUX1  -  cut-like homeobox 1<br>
GFM1  -  g elongation factor, mitochondrial 1<br>
PAIP2B  -  poly(a) binding protein interacting protein 2b<br>
ALYREF  -  aly/ref export factor<br>
PCBP4  -  poly(rc) binding protein 4<br>
ZC3H3  -  zinc finger ccch-type containing 3<br>
HOPX  -  hop homeobox<br>
ZNF74  -  zinc finger protein 74<br>
SF3B3  -  splicing factor 3b, subunit 3, 130kda<br>
ZNF789  -  zinc finger protein 789<br>
LAS1L  -  las1-like (s. cerevisiae)<br>
FXR2  -  fragile x mental retardation, autosomal homolog 2<br>
CCNT1  -  cyclin t1<br>
KTN1  -  kinectin 1 (kinesin receptor)<br>
ZNF41  -  zinc finger protein 41<br>
MEF2A  -  myocyte enhancer factor 2a<br>
ADARB1  -  adenosine deaminase, rna-specific, b1<br>
SAMD4A  -  sterile alpha motif domain containing 4a<br>
MEF2C  -  myocyte enhancer factor 2c<br>
TIPIN  -  timeless interacting protein<br>
TAF12  -  taf12 rna polymerase ii, tata box binding protein (tbp)-associated factor, 20kda<br>
RBM15B  -  rna binding motif protein 15b<br>
ESF1  -  esf1, nucleolar pre-rrna processing protein, homolog (s. cerevisiae)<br>
ANGEL2  -  angel homolog 2 (drosophila)<br>
PABPC4  -  poly(a) binding protein, cytoplasmic 4 (inducible form)<br>
SRPK1  -  srsf protein kinase 1<br>
ZBTB24  -  zinc finger and btb domain containing 24<br>
ROCK2  -  rho-associated, coiled-coil containing protein kinase 2<br>
ZBTB26  -  zinc finger and btb domain containing 26<br>
ZNF558  -  zinc finger protein 558<br>
EIF5B  -  eukaryotic translation initiation factor 5b<br>
KLF10  -  kruppel-like factor 10<br>
SMAD5  -  smad family member 5<br>
</FONT></div>
</td>
</tr>
</table>
<p>Species used: Homo sapiens</p><p>The system has recognized 15881 genes out of 16641 gene terms entered by the user.<br> 15881 genes were recognized by gene symbol and 0 genes by other gene IDs .<br>160 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 15721 genes.<br>Only 15353 of these genes are associated with a GO term.<br></p>
<br><a href='GOFUNCTION.xls'> Output in Microsoft Excel format</a><br>
<p> The GOrilla database is periodically updated using the <a href='http://www.geneontology.org/'>GO database</a> and other sources.<br>
The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 
Jan 26, 2023
). You can bookmark this page and come back to it later.
</p>
<a name='p_value_info'></a><br><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 4519 GO terms.<br>
<a name='fdr_info'></a><br><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br>
<a name='enrich_info'></a><br><b>Enrichment (N, B, n, b)</b> is defined as follows:<br>N - is the total number of genes<br>B - is the total number of genes associated with a specific GO term<br>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br>b - is the number of genes in the intersection<br>Enrichment = (b/n) / (B/N)<br>
<a name='genes_info'></a><br><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br>Each gene name is specified by gene symbol followed by a short description of the gene <br>
<p>
<a href='http://cbl-gorilla.cs.technion.ac.il/'> Back to the GOrilla main page </a>
</p>
</body>
</html>
