<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
<title>
Results
</title>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" >
<script type='text/javascript'>
function toggle(item) {
obj=document.getElementById(item);
visible=(obj.style.display!="none")
key=document.getElementById("ctrl_" + item);
if (visible) {
    obj.style.display="none";
key.innerHTML="[+] Show genes";
}
else {
obj.style.display="block";
key.innerHTML="[-] Hide genes";
}
}
</script>
</head>
<body>
<p><center><b>

</b></center></p>
<table border="1" width="80%" id="tab:pval_legend" align=center>
<CAPTION><EM>P-value color scale</EM></CAPTION>
<tr>
	<td  width=140 align=center> > 10<sup>-3</sup></td>
	<td  width=140 align=center bgcolor='#FFFFCC'> 10<sup>-3</sup> to 10<sup>-5</sup></td>
	<td  width=140 align=center bgcolor='#FFCC66'> 10<sup>-5</sup> to 10<sup>-7</sup></td>
	<td  width=140 align=center bgcolor='#FF9900'> 10<sup>-7</sup> to 10<sup>-9</sup></td>
	<td  width=140 align=center bgcolor='#FF3300'> < 10<sup>-9</sup></td>
</tr >
</table>
<MAP NAME='goPathImage'>
<AREA SHAPE='RECT' COORDS='107,197,197,245' HREF='#0044424' ALT=''>
<AREA SHAPE='RECT' COORDS='53,294,144,357' HREF='#0044446' ALT=''>
<AREA SHAPE='RECT' COORDS='5,197,83,245' HREF='#0044422' ALT=''>
<AREA SHAPE='RECT' COORDS='221,197,355,245' HREF='#0044459' ALT=''>
<AREA SHAPE='RECT' COORDS='252,301,324,349' HREF='#0030315' ALT=''>
<AREA SHAPE='RECT' COORDS='124,101,196,149' HREF='#0044464' ALT=''>
<AREA SHAPE='RECT' COORDS='236,101,321,149' HREF='#0044425' ALT=''>
<AREA SHAPE='RECT' COORDS='160,5,43,29' HREF='#0005575' ALT=''>
</MAP>
<p align="center"><img border="0" src="GOCOMPONENT.png" usemap='#goPathImage' ALT=''></p>
<p>&nbsp;</p>
<table border='1' width='100%' id='table1'>
	<tr>
		<td Align=center ><b>GO term</b></td>
		<td Align=center><b>Description</b></td>
		<td width=120 Align=center><b><a href='#p_value_info'>P-value</a></b></td> 
		<td width=120 Align=center><b><a href='#fdr_info'>FDR q-value</a></b></td> 
		<td width=230 Align=center><b><a href='#enrich_info'> Enrichment (N, B, n, b) </a></b> </td>
		<td Align=center><b><a href='#genes_info'> Genes </a></b></td>
	</tr>
<tr>
<td><a href='http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0044446&amp;view=details' target="_blank">GO:0044446</a><a name=0044446></a></td>
<td>intracellular organelle part</td>
<td  align=center>2.73E-4</td>
<td  align=center>5.26E-1</td>
<td  align=center>1.26 (15353,8308,151,103)</td>
<td>
<a id="ctrl_elements_GO:0044446" href="javascript:toggle('elements_GO:0044446')">[+] Show genes</a><br>
<div id="elements_GO:0044446" style="display:none"> <font size=1>
ACOT13  -  acyl-coa thioesterase 13<br>
CTTN  -  cortactin<br>
REV3L  -  rev3-like, polymerase (dna directed), zeta, catalytic subunit<br>
AKAP9  -  a kinase (prka) anchor protein (yotiao) 9<br>
EHBP1L1  -  eh domain binding protein 1-like 1<br>
ABCA3  -  atp-binding cassette, sub-family a (abc1), member 3<br>
OSBP  -  oxysterol binding protein<br>
RNF14  -  ring finger protein 14<br>
WDR34  -  wd repeat domain 34<br>
CKAP4  -  cytoskeleton-associated protein 4<br>
BLOC1S1  -  biogenesis of lysosomal organelles complex-1, subunit 1<br>
MYO9B  -  myosin ixb<br>
CCDC51  -  coiled-coil domain containing 51<br>
ALYREF  -  aly/ref export factor<br>
HPD  -  4-hydroxyphenylpyruvate dioxygenase<br>
GALNT12  -  udp-n-acetyl-alpha-d-galactosamine:polypeptide n-acetylgalactosaminyltransferase 12 (galnac-t12)<br>
HOPX  -  hop homeobox<br>
ACTC1  -  actin, alpha, cardiac muscle 1<br>
ZNF74  -  zinc finger protein 74<br>
FHL3  -  four and a half lim domains 3<br>
C2orf49  -  chromosome 2 open reading frame 49<br>
ZNF789  -  zinc finger protein 789<br>
LAS1L  -  las1-like (s. cerevisiae)<br>
SAP130  -  sin3a-associated protein, 130kda<br>
FXR2  -  fragile x mental retardation, autosomal homolog 2<br>
DYSF  -  dysferlin, limb girdle muscular dystrophy 2b (autosomal recessive)<br>
KTN1  -  kinectin 1 (kinesin receptor)<br>
MEF2A  -  myocyte enhancer factor 2a<br>
ADARB1  -  adenosine deaminase, rna-specific, b1<br>
SAMD4A  -  sterile alpha motif domain containing 4a<br>
MEF2C  -  myocyte enhancer factor 2c<br>
TTLL11  -  tubulin tyrosine ligase-like family, member 11<br>
GPSM2  -  g-protein signaling modulator 2<br>
TAF12  -  taf12 rna polymerase ii, tata box binding protein (tbp)-associated factor, 20kda<br>
RBM15B  -  rna binding motif protein 15b<br>
PCNT  -  pericentrin<br>
LDB3  -  lim domain binding 3<br>
S100A13  -  s100 calcium binding protein a13<br>
ANGEL2  -  angel homolog 2 (drosophila)<br>
STMN1  -  stathmin 1<br>
SERINC1  -  serine incorporator 1<br>
ROCK2  -  rho-associated, coiled-coil containing protein kinase 2<br>
TPRG1L  -  tumor protein p63 regulated 1-like<br>
CCDC36  -  coiled-coil domain containing 36<br>
IRAK2  -  interleukin-1 receptor-associated kinase 2<br>
SP110  -  sp110 nuclear body protein<br>
ACAA2  -  acetyl-coa acyltransferase 2<br>
BRIX1  -  brx1, biogenesis of ribosomes, homolog (s. cerevisiae)<br>
CAMK2D  -  calcium/calmodulin-dependent protein kinase ii delta<br>
HSPB7  -  heat shock 27kda protein family, member 7 (cardiovascular)<br>
CASQ1  -  calsequestrin 1 (fast-twitch, skeletal muscle)<br>
MAP4  -  microtubule-associated protein 4<br>
MEX3B  -  mex-3 rna binding family member b<br>
KIF1C  -  kinesin family member 1c<br>
FAR1  -  fatty acyl coa reductase 1<br>
ZFPM2  -  zinc finger protein, fog family member 2<br>
THOC7  -  tho complex 7 homolog (drosophila)<br>
ETV6  -  ets variant 6<br>
NFYC  -  nuclear transcription factor y, gamma<br>
VHL  -  von hippel-lindau tumor suppressor, e3 ubiquitin protein ligase<br>
ICMT  -  isoprenylcysteine carboxyl methyltransferase<br>
CUX1  -  cut-like homeobox 1<br>
SYNJ2BP  -  synaptojanin 2 binding protein<br>
GFM1  -  g elongation factor, mitochondrial 1<br>
ALKBH2  -  alkb, alkylation repair homolog 2 (e. coli)<br>
DNAJB4  -  dnaj (hsp40) homolog, subfamily b, member 4<br>
PFKFB2  -  6-phosphofructo-2-kinase/fructose-2,6-biphosphatase 2<br>
ZC3H3  -  zinc finger ccch-type containing 3<br>
CKAP2  -  cytoskeleton associated protein 2<br>
TNFAIP6  -  tumor necrosis factor, alpha-induced protein 6<br>
HSPB6  -  heat shock protein, alpha-crystallin-related, b6<br>
ZFAND2B  -  zinc finger, an1-type domain 2b<br>
ALDH2  -  aldehyde dehydrogenase 2 family (mitochondrial)<br>
MECR  -  mitochondrial trans-2-enoyl-coa reductase<br>
SF3B3  -  splicing factor 3b, subunit 3, 130kda<br>
PPIL1  -  peptidylprolyl isomerase (cyclophilin)-like 1<br>
HS2ST1  -  heparan sulfate 2-o-sulfotransferase 1<br>
CCNT1  -  cyclin t1<br>
GNPAT  -  glyceronephosphate o-acyltransferase<br>
CLN5  -  ceroid-lipofuscinosis, neuronal 5<br>
MYOM1  -  myomesin 1<br>
MID2  -  midline 2<br>
TIPIN  -  timeless interacting protein<br>
MYH7B  -  myosin, heavy chain 7b, cardiac muscle, beta<br>
TGS1  -  trimethylguanosine synthase 1<br>
ZFYVE9  -  zinc finger, fyve domain containing 9<br>
NDUFB10  -  nadh dehydrogenase (ubiquinone) 1 beta subcomplex, 10, 22kda<br>
CLCN4  -  chloride channel, voltage-sensitive 4<br>
ESF1  -  esf1, nucleolar pre-rrna processing protein, homolog (s. cerevisiae)<br>
SRPK1  -  srsf protein kinase 1<br>
BIN1  -  bridging integrator 1<br>
PLCE1  -  phospholipase c, epsilon 1<br>
ZBTB26  -  zinc finger and btb domain containing 26<br>
CKMT2  -  creatine kinase, mitochondrial 2 (sarcomeric)<br>
MAP7D1  -  map7 domain containing 1<br>
RBX1  -  ring-box 1, e3 ubiquitin protein ligase<br>
CUL4A  -  cullin 4a<br>
CWF19L1  -  cwf19-like 1, cell cycle control (s. pombe)<br>
KLF10  -  kruppel-like factor 10<br>
LIPT1  -  lipoyltransferase 1<br>
SMAD5  -  smad family member 5<br>
ITSN1  -  intersectin 1 (sh3 domain protein)<br>
USP13  -  ubiquitin specific peptidase 13 (isopeptidase t-3)<br>
</FONT></div>
</td>
</tr>
<tr>
<td><a href='http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0030315&amp;view=details' target="_blank">GO:0030315</a><a name=0030315></a></td>
<td>T-tubule</td>
<td  align=center>7.53E-4</td>
<td  align=center>7.26E-1</td>
<td  align=center>9.68 (15353,42,151,4)</td>
<td>
<a id="ctrl_elements_GO:0030315" href="javascript:toggle('elements_GO:0030315')">[+] Show genes</a><br>
<div id="elements_GO:0030315" style="display:none"> <font size=1>
CASQ1  -  calsequestrin 1 (fast-twitch, skeletal muscle)<br>
SCN1B  -  sodium channel, voltage-gated, type i, beta subunit<br>
DYSF  -  dysferlin, limb girdle muscular dystrophy 2b (autosomal recessive)<br>
BIN1  -  bridging integrator 1<br>
</FONT></div>
</td>
</tr>
</table>
<p>Species used: Homo sapiens</p><p>The system has recognized 15881 genes out of 16641 gene terms entered by the user.<br> 15881 genes were recognized by gene symbol and 0 genes by other gene IDs .<br>160 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 15721 genes.<br>Only 15353 of these genes are associated with a GO term.<br></p>
<br><a href='GOCOMPONENT.xls'> Output in Microsoft Excel format</a><br>
<p> The GOrilla database is periodically updated using the <a href='http://www.geneontology.org/'>GO database</a> and other sources.<br>
The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 
Jan 26, 2023
). You can bookmark this page and come back to it later.
</p>
<a name='p_value_info'></a><br><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 1928 GO terms.<br>
<a name='fdr_info'></a><br><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br>
<a name='enrich_info'></a><br><b>Enrichment (N, B, n, b)</b> is defined as follows:<br>N - is the total number of genes<br>B - is the total number of genes associated with a specific GO term<br>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br>b - is the number of genes in the intersection<br>Enrichment = (b/n) / (B/N)<br>
<a name='genes_info'></a><br><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br>Each gene name is specified by gene symbol followed by a short description of the gene <br>
<p>
<a href='http://cbl-gorilla.cs.technion.ac.il/'> Back to the GOrilla main page </a>
</p>
</body>
</html>
