<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>
<title>

Results

</title>
<meta content="text/html;charset=utf-8" http-equiv="Content-Type"/>
<script type="text/javascript">

function toggle(item) {

obj=document.getElementById(item);

visible=(obj.style.display!="none")

key=document.getElementById("ctrl_" + item);

if (visible) {

    obj.style.display="none";

key.innerHTML="[+] Show genes";

}

else {

obj.style.display="block";

key.innerHTML="[-] Hide genes";

}

}

</script>
</head>
<body>
<p><center><b>
</b></center></p>
<table align="center" border="1" id="tab:pval_legend" width="80%">
<caption><em>P-value color scale</em></caption>
<tr>
<td align="center" width="140"> &gt; 10<sup>-3</sup></td>
<td align="center" bgcolor="#FFFFCC" width="140"> 10<sup>-3</sup> to 10<sup>-5</sup></td>
<td align="center" bgcolor="#FFCC66" width="140"> 10<sup>-5</sup> to 10<sup>-7</sup></td>
<td align="center" bgcolor="#FF9900" width="140"> 10<sup>-7</sup> to 10<sup>-9</sup></td>
<td align="center" bgcolor="#FF3300" width="140"> &lt; 10<sup>-9</sup></td>
</tr>
</table>
<map name="goPathImage">
<area alt="" coords="215,301,287,349" href="#0006810" shape="RECT"/>
<area alt="" coords="205,405,296,453" href="#0046907" shape="RECT"/>
<area alt="" coords="28,197,103,245" href="#0007624" shape="RECT"/>
<area alt="" coords="105,301,191,349" href="#0051641" shape="RECT"/>
<area alt="" coords="28,101,103,149" href="#0048511" shape="RECT"/>
<area alt="" coords="311,294,417,357" href="#0051649" shape="RECT"/>
<area alt="" coords="127,101,212,149" href="#0051179" shape="RECT"/>
<area alt="" coords="197,197,304,245" href="#0051234" shape="RECT"/>
<area alt="" coords="117,5,5,29" href="#0008150" shape="RECT"/>
</map>
<p align="center"><img alt="" border="0" src="/static/analysis/data5/go/39/GOPROCESS.png" usemap="#goPathImage"/></p>
<p>Â </p>
<table border="1" id="table1" width="100%">
<tr>
<td align="center"><b>GO term</b></td>
<td align="center"><b>Description</b></td>
<td align="center" width="120"><b><a href="#p_value_info">P-value</a></b></td>
<td align="center" width="120"><b><a href="#fdr_info">FDR q-value</a></b></td>
<td align="center" width="230"><b><a href="#enrich_info"> Enrichment (N, B, n, b) </a></b> </td>
<td align="center"><b><a href="#genes_info"> Genes </a></b></td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0007624&amp;view=details" target="_blank">GO:0007624</a><a name="0007624"></a></td>
<td>ultradian rhythm</td>
<td align="center">1.77E-4</td>
<td align="center">1E0</td>
<td align="center">74.89 (15353,2,205,2)</td>
<td>
<a href="javascript:toggle('elements_GO:0007624')" id="ctrl_elements_GO:0007624">[+] Show genes</a><br/>
<div id="elements_GO:0007624" style="display:none"> <font size="1">

ENOX1  -  ecto-nox disulfide-thiol exchanger 1<br/>

ENOX2  -  ecto-nox disulfide-thiol exchanger 2<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0046907&amp;view=details" target="_blank">GO:0046907</a><a name="0046907"></a></td>
<td>intracellular transport</td>
<td align="center">3.88E-4</td>
<td align="center">1E0</td>
<td align="center">1.90 (15353,1221,205,31)</td>
<td>
<a href="javascript:toggle('elements_GO:0046907')" id="ctrl_elements_GO:0046907">[+] Show genes</a><br/>
<div id="elements_GO:0046907" style="display:none"> <font size="1">

RAB28  -  rab28, member ras oncogene family<br/>

SNX3  -  sorting nexin 3<br/>

TUBA1B  -  tubulin, alpha 1b<br/>

PTTG1IP  -  pituitary tumor-transforming 1 interacting protein<br/>

ATXN1  -  ataxin 1<br/>

VAPB  -  vamp (vesicle-associated membrane protein)-associated protein b and c<br/>

LDLR  -  low density lipoprotein receptor<br/>

PEX2  -  peroxisomal biogenesis factor 2<br/>

DYNC2H1  -  dynein, cytoplasmic 2, heavy chain 1<br/>

WDR34  -  wd repeat domain 34<br/>

RABGEF1  -  rab guanine nucleotide exchange factor (gef) 1<br/>

SPCS3  -  signal peptidase complex subunit 3 homolog (s. cerevisiae)<br/>

RIMS3  -  regulating synaptic membrane exocytosis 3<br/>

SEC31A  -  sec31 homolog a (s. cerevisiae)<br/>

RNF157  -  ring finger protein 157<br/>

CEP120  -  centrosomal protein 120kda<br/>

TIMM9  -  translocase of inner mitochondrial membrane 9 homolog (yeast)<br/>

RAB8A  -  rab8a, member ras oncogene family<br/>

VPS37D  -  vacuolar protein sorting 37 homolog d (s. cerevisiae)<br/>

CADPS2  -  ca++-dependent secretion activator 2<br/>

PCM1  -  pericentriolar material 1<br/>

BACH2  -  btb and cnc homology 1, basic leucine zipper transcription factor 2<br/>

KPNA2  -  karyopherin alpha 2 (rag cohort 1, importin alpha 1)<br/>

IPO4  -  importin 4<br/>

MYO1G  -  myosin ig<br/>

BTBD8  -  btb (poz) domain containing 8<br/>

RAB5C  -  rab5c, member ras oncogene family<br/>

NUP153  -  nucleoporin 153kda<br/>

EHD2  -  eh-domain containing 2<br/>

SNX1  -  sorting nexin 1<br/>

EHD3  -  eh-domain containing 3<br/>
</font></div>
</td>
</tr>
</table>
<p>Species used: Homo sapiens</p><p>The system has recognized 15936 genes out of 16698 gene terms entered by the user.<br/> 15936 genes were recognized by gene symbol and 0 genes by other gene IDs .<br/>215 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 15721 genes.<br/>Only 15353 of these genes are associated with a GO term.<br/></p>
<br/><a href="GOPROCESS.xls"> Output in Microsoft Excel format</a><br/>
<p> The GOrilla database is periodically updated using the <a href="http://www.geneontology.org/">GO database</a> and other sources.<br/>

The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 

Jan 26, 2023

). You can bookmark this page and come back to it later.

</p>
<a name="p_value_info"></a><br/><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 15044 GO terms.<br/>
<a name="fdr_info"></a><br/><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br/>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br/>
<a name="enrich_info"></a><br/><b>Enrichment (N, B, n, b)</b> is defined as follows:<br/>N - is the total number of genes<br/>B - is the total number of genes associated with a specific GO term<br/>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br/>b - is the number of genes in the intersection<br/>Enrichment = (b/n) / (B/N)<br/>
<a name="genes_info"></a><br/><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br/>Each gene name is specified by gene symbol followed by a short description of the gene <br/>
<p>
<a href="http://cbl-gorilla.cs.technion.ac.il/"> Back to the GOrilla main page </a>
</p>
</body>
</html>
