<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>
<title>

Results

</title>
<meta content="text/html;charset=utf-8" http-equiv="Content-Type"/>
<script type="text/javascript">

function toggle(item) {

obj=document.getElementById(item);

visible=(obj.style.display!="none")

key=document.getElementById("ctrl_" + item);

if (visible) {

    obj.style.display="none";

key.innerHTML="[+] Show genes";

}

else {

obj.style.display="block";

key.innerHTML="[-] Hide genes";

}

}

</script>
</head>
<body>
<p><center><b>
</b></center></p>
<table align="center" border="1" id="tab:pval_legend" width="80%">
<caption><em>P-value color scale</em></caption>
<tr>
<td align="center" width="140"> &gt; 10<sup>-3</sup></td>
<td align="center" bgcolor="#FFFFCC" width="140"> 10<sup>-3</sup> to 10<sup>-5</sup></td>
<td align="center" bgcolor="#FFCC66" width="140"> 10<sup>-5</sup> to 10<sup>-7</sup></td>
<td align="center" bgcolor="#FF9900" width="140"> 10<sup>-7</sup> to 10<sup>-9</sup></td>
<td align="center" bgcolor="#FF3300" width="140"> &lt; 10<sup>-9</sup></td>
</tr>
</table>
<map name="goPathImage">
<area alt="" coords="5,325,120,405" href="#0004888" shape="RECT"/>
<area alt="" coords="20,102,105,165" href="#0060089" shape="RECT"/>
<area alt="" coords="24,214,101,277" href="#0038023" shape="RECT"/>
<area alt="" coords="177,5,64,29" href="#0003674" shape="RECT"/>
<area alt="" coords="129,102,225,165" href="#0140110" shape="RECT"/>
<area alt="" coords="249,109,321,157" href="#0005488" shape="RECT"/>
<area alt="" coords="232,221,339,269" href="#0005515" shape="RECT"/>
<area alt="" coords="229,341,341,389" href="#0019899" shape="RECT"/>
<area alt="" coords="233,453,337,501" href="#0019900" shape="RECT"/>
<area alt="" coords="232,549,339,597" href="#0019901" shape="RECT"/>
<area alt="" coords="232,645,339,693" href="#0043422" shape="RECT"/>
</map>
<p align="center"><img alt="" border="0" src="/static/analysis/data1/go/32/GOFUNCTION.png" usemap="#goPathImage"/></p>
<p>Â </p>
<table border="1" id="table1" width="100%">
<tr>
<td align="center"><b>GO term</b></td>
<td align="center"><b>Description</b></td>
<td align="center" width="120"><b><a href="#p_value_info">P-value</a></b></td>
<td align="center" width="120"><b><a href="#fdr_info">FDR q-value</a></b></td>
<td align="center" width="230"><b><a href="#enrich_info"> Enrichment (N, B, n, b) </a></b> </td>
<td align="center"><b><a href="#genes_info"> Genes </a></b></td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0060089&amp;view=details" target="_blank">GO:0060089</a><a name="0060089"></a></td>
<td>molecular transducer activity</td>
<td align="center">2.92E-4</td>
<td align="center">1E0</td>
<td align="center">2.70 (8795,225,232,16)</td>
<td>
<a href="javascript:toggle('elements_GO:0060089')" id="ctrl_elements_GO:0060089">[+] Show genes</a><br/>
<div id="elements_GO:0060089" style="display:none"> <font size="1">

P2RY10  -  purinergic receptor p2y, g-protein coupled, 10<br/>

PRKAA1  -  protein kinase, amp-activated, alpha 1 catalytic subunit<br/>

CD46  -  cd46 molecule, complement regulatory protein<br/>

CCR6  -  chemokine (c-c motif) receptor 6<br/>

GPR65  -  g protein-coupled receptor 65<br/>

HLA-DOB  -  major histocompatibility complex, class ii, do beta<br/>

CDK13  -  cyclin-dependent kinase 13<br/>

CCR1  -  chemokine (c-c motif) receptor 1<br/>

FCRL5  -  fc receptor-like 5<br/>

LAG3  -  lymphocyte-activation gene 3<br/>

CD69  -  cd69 molecule<br/>

TM2D1  -  tm2 domain containing 1<br/>

SEC62  -  sec62 homolog (s. cerevisiae)<br/>

INTS6  -  integrator complex subunit 6<br/>

BMPR2  -  bone morphogenetic protein receptor, type ii (serine/threonine kinase)<br/>

CD79B  -  cd79b molecule, immunoglobulin-associated beta<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0004888&amp;view=details" target="_blank">GO:0004888</a><a name="0004888"></a></td>
<td>transmembrane signaling receptor activity</td>
<td align="center">4.36E-4</td>
<td align="center">8.01E-1</td>
<td align="center">3.14 (8795,145,232,12)</td>
<td>
<a href="javascript:toggle('elements_GO:0004888')" id="ctrl_elements_GO:0004888">[+] Show genes</a><br/>
<div id="elements_GO:0004888" style="display:none"> <font size="1">

LAG3  -  lymphocyte-activation gene 3<br/>

FCRL5  -  fc receptor-like 5<br/>

CD69  -  cd69 molecule<br/>

P2RY10  -  purinergic receptor p2y, g-protein coupled, 10<br/>

TM2D1  -  tm2 domain containing 1<br/>

CCR6  -  chemokine (c-c motif) receptor 6<br/>

GPR65  -  g protein-coupled receptor 65<br/>

INTS6  -  integrator complex subunit 6<br/>

HLA-DOB  -  major histocompatibility complex, class ii, do beta<br/>

BMPR2  -  bone morphogenetic protein receptor, type ii (serine/threonine kinase)<br/>

CD79B  -  cd79b molecule, immunoglobulin-associated beta<br/>

CCR1  -  chemokine (c-c motif) receptor 1<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0038023&amp;view=details" target="_blank">GO:0038023</a><a name="0038023"></a></td>
<td>signaling receptor activity</td>
<td align="center">7.74E-4</td>
<td align="center">9.48E-1</td>
<td align="center">2.67 (8795,199,232,14)</td>
<td>
<a href="javascript:toggle('elements_GO:0038023')" id="ctrl_elements_GO:0038023">[+] Show genes</a><br/>
<div id="elements_GO:0038023" style="display:none"> <font size="1">

P2RY10  -  purinergic receptor p2y, g-protein coupled, 10<br/>

CD46  -  cd46 molecule, complement regulatory protein<br/>

CCR6  -  chemokine (c-c motif) receptor 6<br/>

GPR65  -  g protein-coupled receptor 65<br/>

HLA-DOB  -  major histocompatibility complex, class ii, do beta<br/>

CCR1  -  chemokine (c-c motif) receptor 1<br/>

FCRL5  -  fc receptor-like 5<br/>

LAG3  -  lymphocyte-activation gene 3<br/>

CD69  -  cd69 molecule<br/>

TM2D1  -  tm2 domain containing 1<br/>

SEC62  -  sec62 homolog (s. cerevisiae)<br/>

INTS6  -  integrator complex subunit 6<br/>

BMPR2  -  bone morphogenetic protein receptor, type ii (serine/threonine kinase)<br/>

CD79B  -  cd79b molecule, immunoglobulin-associated beta<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0140110&amp;view=details" target="_blank">GO:0140110</a><a name="0140110"></a></td>
<td>transcription regulator activity</td>
<td align="center">7.84E-4</td>
<td align="center">7.2E-1</td>
<td align="center">1.67 (8795,910,232,40)</td>
<td>
<a href="javascript:toggle('elements_GO:0140110')" id="ctrl_elements_GO:0140110">[+] Show genes</a><br/>
<div id="elements_GO:0140110" style="display:none"> <font size="1">

MEF2B  -  myocyte enhancer factor 2b<br/>

MGA  -  mga, max dimerization protein<br/>

IFI16  -  interferon, gamma-inducible protein 16<br/>

IRF9  -  interferon regulatory factor 9<br/>

MED17  -  mediator complex subunit 17<br/>

ELMSAN1  -  elm2 and myb/sant-like domain containing 1<br/>

LPIN1  -  lipin 1<br/>

ETV7  -  ets variant 7<br/>

RELA  -  v-rel avian reticuloendotheliosis viral oncogene homolog a<br/>

SIAH2  -  siah e3 ubiquitin protein ligase 2<br/>

MKL2  -  mkl/myocardin-like 2<br/>

EP300  -  e1a binding protein p300<br/>

ZNF518A  -  zinc finger protein 518a<br/>

SPI1  -  spleen focus forming virus (sffv) proviral integration oncogene<br/>

HKR1  -  hkr1, gli-kruppel zinc finger family member<br/>

DDX17  -  dead (asp-glu-ala-asp) box helicase 17<br/>

PARP9  -  poly (adp-ribose) polymerase family, member 9<br/>

SUPT20H  -  suppressor of ty 20 homolog (s. cerevisiae)<br/>

BCL11A  -  b-cell cll/lymphoma 11a (zinc finger protein)<br/>

ZNF691  -  zinc finger protein 691<br/>

CREBRF  -  creb3 regulatory factor<br/>

NR1H2  -  nuclear receptor subfamily 1, group h, member 2<br/>

MED13L  -  mediator complex subunit 13-like<br/>

CIR1  -  corepressor interacting with rbpj, 1<br/>

ZBTB14  -  zinc finger and btb domain containing 14<br/>

ZBTB24  -  zinc finger and btb domain containing 24<br/>

ZNF302  -  zinc finger protein 302<br/>

MED16  -  mediator complex subunit 16<br/>

TRIM38  -  tripartite motif containing 38<br/>

BCL10  -  b-cell cll/lymphoma 10<br/>

IRF2  -  interferon regulatory factor 2<br/>

ZNF226  -  zinc finger protein 226<br/>

ELF1  -  e74-like factor 1 (ets domain transcription factor)<br/>

PARP14  -  poly (adp-ribose) polymerase family, member 14<br/>

TADA2B  -  transcriptional adaptor 2b<br/>

CREBBP  -  creb binding protein<br/>

MED19  -  mediator complex subunit 19<br/>

ZNF791  -  zinc finger protein 791<br/>

ZNF790  -  zinc finger protein 790<br/>

ZNF394  -  zinc finger protein 394<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0043422&amp;view=details" target="_blank">GO:0043422</a><a name="0043422"></a></td>
<td>protein kinase B binding</td>
<td align="center">9.2E-4</td>
<td align="center">6.76E-1</td>
<td align="center">14.22 (8795,8,232,3)</td>
<td>
<a href="javascript:toggle('elements_GO:0043422')" id="ctrl_elements_GO:0043422">[+] Show genes</a><br/>
<div id="elements_GO:0043422" style="display:none"> <font size="1">

TRAF6  -  tnf receptor-associated factor 6, e3 ubiquitin protein ligase<br/>

SRSF5  -  serine/arginine-rich splicing factor 5<br/>

BCL10  -  b-cell cll/lymphoma 10<br/>
</font></div>
</td>
</tr>
</table>
<p>Species used: Homo sapiens</p><p>The system has recognized 9268 genes out of 9724 gene terms entered by the user.<br/> 9268 genes were recognized by gene symbol and 0 genes by other gene IDs .<br/>240 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 9028 genes.<br/>Only 8795 of these genes are associated with a GO term.<br/></p>
<br/><a href="GOFUNCTION.xls"> Output in Microsoft Excel format</a><br/>
<p> The GOrilla database is periodically updated using the <a href="http://www.geneontology.org/">GO database</a> and other sources.<br/>

The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 

Jan 16, 2023

). You can bookmark this page and come back to it later.

</p>
<a name="p_value_info"></a><br/><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 3672 GO terms.<br/>
<a name="fdr_info"></a><br/><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br/>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br/>
<a name="enrich_info"></a><br/><b>Enrichment (N, B, n, b)</b> is defined as follows:<br/>N - is the total number of genes<br/>B - is the total number of genes associated with a specific GO term<br/>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br/>b - is the number of genes in the intersection<br/>Enrichment = (b/n) / (B/N)<br/>
<a name="genes_info"></a><br/><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br/>Each gene name is specified by gene symbol followed by a short description of the gene <br/>
<p>
<a href="http://cbl-gorilla.cs.technion.ac.il/"> Back to the GOrilla main page </a>
</p>
</body>
</html>
