<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>
<title>

Results

</title>
<meta content="text/html;charset=utf-8" http-equiv="Content-Type"/>
<script type="text/javascript">

function toggle(item) {

obj=document.getElementById(item);

visible=(obj.style.display!="none")

key=document.getElementById("ctrl_" + item);

if (visible) {

    obj.style.display="none";

key.innerHTML="[+] Show genes";

}

else {

obj.style.display="block";

key.innerHTML="[-] Hide genes";

}

}

</script>
</head>
<body>
<p><center><b>
</b></center></p>
<table align="center" border="1" id="tab:pval_legend" width="80%">
<caption><em>P-value color scale</em></caption>
<tr>
<td align="center" width="140"> &gt; 10<sup>-3</sup></td>
<td align="center" bgcolor="#FFFFCC" width="140"> 10<sup>-3</sup> to 10<sup>-5</sup></td>
<td align="center" bgcolor="#FFCC66" width="140"> 10<sup>-5</sup> to 10<sup>-7</sup></td>
<td align="center" bgcolor="#FF9900" width="140"> 10<sup>-7</sup> to 10<sup>-9</sup></td>
<td align="center" bgcolor="#FF3300" width="140"> &lt; 10<sup>-9</sup></td>
</tr>
</table>
<map name="goPathImage">
<area alt="" coords="539,197,635,245" href="#0005615" shape="RECT"/>
<area alt="" coords="395,293,528,373" href="#0120038" shape="RECT"/>
<area alt="" coords="329,421,436,469" href="#0031253" shape="RECT"/>
<area alt="" coords="9,101,137,149" href="#0032991" shape="RECT"/>
<area alt="" coords="5,197,120,245" href="#0019814" shape="RECT"/>
<area alt="" coords="304,5,187,29" href="#0005575" shape="RECT"/>
<area alt="" coords="144,197,227,245" href="#0016020" shape="RECT"/>
<area alt="" coords="371,101,443,149" href="#0044464" shape="RECT"/>
<area alt="" coords="261,101,347,149" href="#0044425" shape="RECT"/>
<area alt="" coords="521,101,617,149" href="#0044421" shape="RECT"/>
<area alt="" coords="237,309,371,357" href="#0098590" shape="RECT"/>
<area alt="" coords="408,197,515,245" href="#0044463" shape="RECT"/>
<area alt="" coords="333,517,432,565" href="#0031526" shape="RECT"/>
<area alt="" coords="251,197,384,245" href="#0044459" shape="RECT"/>
</map>
<p align="center"><img alt="" border="0" src="/static/analysis/data1/go/36/GOCOMPONENT.png" usemap="#goPathImage"/></p>
<p>Â </p>
<table border="1" id="table1" width="100%">
<tr>
<td align="center"><b>GO term</b></td>
<td align="center"><b>Description</b></td>
<td align="center" width="120"><b><a href="#p_value_info">P-value</a></b></td>
<td align="center" width="120"><b><a href="#fdr_info">FDR q-value</a></b></td>
<td align="center" width="230"><b><a href="#enrich_info"> Enrichment (N, B, n, b) </a></b> </td>
<td align="center"><b><a href="#genes_info"> Genes </a></b></td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0019814&amp;view=details" target="_blank">GO:0019814</a><a name="0019814"></a></td>
<td>immunoglobulin complex</td>
<td align="center">3.38E-4</td>
<td align="center">5.81E-1</td>
<td align="center">7.98 (8795,26,212,5)</td>
<td>
<a href="javascript:toggle('elements_GO:0019814')" id="ctrl_elements_GO:0019814">[+] Show genes</a><br/>
<div id="elements_GO:0019814" style="display:none"> <font size="1">

IGHM  -  immunoglobulin heavy constant mu<br/>

IGKV1-12  -  immunoglobulin kappa variable 1-12<br/>

IGKC  -  immunoglobulin kappa constant<br/>

IGHD  -  immunoglobulin heavy constant delta<br/>

CD79B  -  cd79b molecule, immunoglobulin-associated beta<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0005615&amp;view=details" target="_blank">GO:0005615</a><a name="0005615"></a></td>
<td>extracellular space</td>
<td align="center">3.76E-4</td>
<td align="center">3.23E-1</td>
<td align="center">2.39 (8795,330,212,19)</td>
<td>
<a href="javascript:toggle('elements_GO:0005615')" id="ctrl_elements_GO:0005615">[+] Show genes</a><br/>
<div id="elements_GO:0005615" style="display:none"> <font size="1">

IGHM  -  immunoglobulin heavy constant mu<br/>

CNOT1  -  ccr4-not transcription complex, subunit 1<br/>

LRIG2  -  leucine-rich repeats and immunoglobulin-like domains 2<br/>

OAS3  -  2'-5'-oligoadenylate synthetase 3, 100kda<br/>

IGKC  -  immunoglobulin kappa constant<br/>

RNASET2  -  ribonuclease t2<br/>

SERPINB9  -  serpin peptidase inhibitor, clade b (ovalbumin), member 9<br/>

PRR4  -  proline rich 4 (lacrimal)<br/>

CTSO  -  cathepsin o<br/>

UBC  -  ubiquitin c<br/>

APOL1  -  apolipoprotein l, 1<br/>

MECP2  -  methyl cpg binding protein 2 (rett syndrome)<br/>

LRRK2  -  leucine-rich repeat kinase 2<br/>

IGKV1-12  -  immunoglobulin kappa variable 1-12<br/>

TNFSF12  -  tumor necrosis factor (ligand) superfamily, member 12<br/>

PIBF1  -  progesterone immunomodulatory binding factor 1<br/>

BMPR2  -  bone morphogenetic protein receptor, type ii (serine/threonine kinase)<br/>

MOV10  -  mov10, moloney leukemia virus 10, homolog (mouse)<br/>

HIST1H2BG  -  histone cluster 1, h2bg<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0031526&amp;view=details" target="_blank">GO:0031526</a><a name="0031526"></a></td>
<td>brush border membrane</td>
<td align="center">7.7E-4</td>
<td align="center">4.42E-1</td>
<td align="center">9.22 (8795,18,212,4)</td>
<td>
<a href="javascript:toggle('elements_GO:0031526')" id="ctrl_elements_GO:0031526">[+] Show genes</a><br/>
<div id="elements_GO:0031526" style="display:none"> <font size="1">

SLC11A2  -  solute carrier family 11 (proton-coupled divalent metal ion transporter), member 2<br/>

KCNK1  -  potassium channel, subfamily k, member 1<br/>

ATP7A  -  atpase, cu++ transporting, alpha polypeptide<br/>

GNA13  -  guanine nucleotide binding protein (g protein), alpha 13<br/>
</font></div>
</td>
</tr>
</table>
<p>Species used: Homo sapiens</p><p>The system has recognized 9247 genes out of 9703 gene terms entered by the user.<br/> 9247 genes were recognized by gene symbol and 0 genes by other gene IDs .<br/>219 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 9028 genes.<br/>Only 8795 of these genes are associated with a GO term.<br/></p>
<br/><a href="GOCOMPONENT.xls"> Output in Microsoft Excel format</a><br/>
<p> The GOrilla database is periodically updated using the <a href="http://www.geneontology.org/">GO database</a> and other sources.<br/>

The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 

Jan 16, 2023

). You can bookmark this page and come back to it later.

</p>
<a name="p_value_info"></a><br/><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 1721 GO terms.<br/>
<a name="fdr_info"></a><br/><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br/>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br/>
<a name="enrich_info"></a><br/><b>Enrichment (N, B, n, b)</b> is defined as follows:<br/>N - is the total number of genes<br/>B - is the total number of genes associated with a specific GO term<br/>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br/>b - is the number of genes in the intersection<br/>Enrichment = (b/n) / (B/N)<br/>
<a name="genes_info"></a><br/><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br/>Each gene name is specified by gene symbol followed by a short description of the gene <br/>
<p>
<a href="http://cbl-gorilla.cs.technion.ac.il/"> Back to the GOrilla main page </a>
</p>
</body>
</html>
