<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>
<title>

Results

</title>
<meta content="text/html;charset=utf-8" http-equiv="Content-Type"/>
<script type="text/javascript">

function toggle(item) {

obj=document.getElementById(item);

visible=(obj.style.display!="none")

key=document.getElementById("ctrl_" + item);

if (visible) {

    obj.style.display="none";

key.innerHTML="[+] Show genes";

}

else {

obj.style.display="block";

key.innerHTML="[-] Hide genes";

}

}

</script>
</head>
<body>
<p><center><b>
</b></center></p>
<table align="center" border="1" id="tab:pval_legend" width="80%">
<caption><em>P-value color scale</em></caption>
<tr>
<td align="center" width="140"> &gt; 10<sup>-3</sup></td>
<td align="center" bgcolor="#FFFFCC" width="140"> 10<sup>-3</sup> to 10<sup>-5</sup></td>
<td align="center" bgcolor="#FFCC66" width="140"> 10<sup>-5</sup> to 10<sup>-7</sup></td>
<td align="center" bgcolor="#FF9900" width="140"> 10<sup>-7</sup> to 10<sup>-9</sup></td>
<td align="center" bgcolor="#FF3300" width="140"> &lt; 10<sup>-9</sup></td>
</tr>
</table>
<map name="goPathImage">
<area alt="" coords="29,704,205,752" href="#0017111" shape="RECT"/>
<area alt="" coords="21,833,133,881" href="#0016887" shape="RECT"/>
<area alt="" coords="415,213,540,261" href="#0030234" shape="RECT"/>
<area alt="" coords="317,471,437,519" href="#0004857" shape="RECT"/>
<area alt="" coords="629,318,728,381" href="#0019208" shape="RECT"/>
<area alt="" coords="421,318,597,381" href="#0060589" shape="RECT"/>
<area alt="" coords="543,809,695,906" href="#0015318" shape="RECT"/>
<area alt="" coords="476,1083,631,1163" href="#0022890" shape="RECT"/>
<area alt="" coords="121,213,204,261" href="#0016787" shape="RECT"/>
<area alt="" coords="81,309,193,389" href="#0016817" shape="RECT"/>
<area alt="" coords="391,600,511,648" href="#0042030" shape="RECT"/>
<area alt="" coords="256,1357,392,1437" href="#0042625" shape="RECT"/>
<area alt="" coords="259,1597,384,1712" href="#0044769" shape="RECT"/>
<area alt="" coords="359,1493,492,1541" href="#0019829" shape="RECT"/>
<area alt="" coords="611,463,752,526" href="#0019888" shape="RECT"/>
<area alt="" coords="169,826,321,889" href="#0022804" shape="RECT"/>
<area alt="" coords="169,955,284,1035" href="#0015399" shape="RECT"/>
<area alt="" coords="308,955,423,1035" href="#0022853" shape="RECT"/>
<area alt="" coords="400,1760,533,1840" href="#0046961" shape="RECT"/>
<area alt="" coords="149,1083,328,1163" href="#0015405" shape="RECT"/>
<area alt="" coords="179,1211,299,1309" href="#0042626" shape="RECT"/>
<area alt="" coords="408,1631,525,1679" href="#0036442" shape="RECT"/>
<area alt="" coords="516,1486,671,1549" href="#0015078" shape="RECT"/>
<area alt="" coords="236,109,324,157" href="#0005215" shape="RECT"/>
<area alt="" coords="309,697,424,759" href="#0022857" shape="RECT"/>
<area alt="" coords="60,600,185,648" href="#0016462" shape="RECT"/>
<area alt="" coords="133,109,208,157" href="#0003824" shape="RECT"/>
<area alt="" coords="505,1211,660,1309" href="#0015077" shape="RECT"/>
<area alt="" coords="461,471,587,519" href="#0060590" shape="RECT"/>
<area alt="" coords="447,963,601,1026" href="#0008324" shape="RECT"/>
<area alt="" coords="377,102,460,165" href="#0098772" shape="RECT"/>
<area alt="" coords="49,437,220,552" href="#0016818" shape="RECT"/>
<area alt="" coords="345,826,481,889" href="#0015075" shape="RECT"/>
<area alt="" coords="15,971,135,1019" href="#0042623" shape="RECT"/>
<area alt="" coords="5,1083,125,1163" href="#0043492" shape="RECT"/>
<area alt="" coords="280,5,166,29" href="#0003674" shape="RECT"/>
</map>
<p align="center"><img alt="" border="0" src="/static/analysis/data1/go/26/GOFUNCTION.png" usemap="#goPathImage"/></p>
<p>Â </p>
<table border="1" id="table1" width="100%">
<tr>
<td align="center"><b>GO term</b></td>
<td align="center"><b>Description</b></td>
<td align="center" width="120"><b><a href="#p_value_info">P-value</a></b></td>
<td align="center" width="120"><b><a href="#fdr_info">FDR q-value</a></b></td>
<td align="center" width="230"><b><a href="#enrich_info"> Enrichment (N, B, n, b) </a></b> </td>
<td align="center"><b><a href="#genes_info"> Genes </a></b></td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0030234&amp;view=details" target="_blank">GO:0030234</a><a name="0030234"></a></td>
<td>enzyme regulator activity</td>
<td align="center">1.04E-4</td>
<td align="center">3.8E-1</td>
<td align="center">1.66 (8795,578,514,56)</td>
<td>
<a href="javascript:toggle('elements_GO:0030234')" id="ctrl_elements_GO:0030234">[+] Show genes</a><br/>
<div id="elements_GO:0030234" style="display:none"> <font size="1">

PSMD6  -  proteasome (prosome, macropain) 26s subunit, non-atpase, 6<br/>

NCF4  -  neutrophil cytosolic factor 4, 40kda<br/>

DDX3X  -  dead (asp-glu-ala-asp) box helicase 3, x-linked<br/>

DTX3L  -  deltex 3-like (drosophila)<br/>

SIPA1L3  -  signal-induced proliferation-associated 1 like 3<br/>

ARHGAP15  -  rho gtpase activating protein 15<br/>

LAMTOR3  -  late endosomal/lysosomal adaptor, mapk and mtor activator 3<br/>

PHACTR4  -  phosphatase and actin regulator 4<br/>

TBC1D2B  -  tbc1 domain family, member 2b<br/>

ARHGAP10  -  rho gtpase activating protein 10<br/>

PIK3R3  -  phosphoinositide-3-kinase, regulatory subunit 3 (gamma)<br/>

FAM13B  -  family with sequence similarity 13, member b<br/>

DYNLL1  -  dynein, light chain, lc8-type 1<br/>

TBC1D10A  -  tbc1 domain family, member 10a<br/>

TANK  -  traf family member-associated nfkb activator<br/>

PPP4R2  -  protein phosphatase 4, regulatory subunit 2<br/>

RASGRP3  -  ras guanyl releasing protein 3 (calcium and dag-regulated)<br/>

RAB3GAP1  -  rab3 gtpase activating protein subunit 1 (catalytic)<br/>

MYO9B  -  myosin ixb<br/>

APBA3  -  amyloid beta (a4) precursor protein-binding, family a, member 3<br/>

CTSH  -  cathepsin h<br/>

ATPIF1  -  atpase inhibitory factor 1<br/>

ARL2BP  -  adp-ribosylation factor-like 2 binding protein<br/>

DCP1A  -  decapping mrna 1a<br/>

PPP2R2A  -  protein phosphatase 2, regulatory subunit b, alpha<br/>

ANKLE2  -  ankyrin repeat and lem domain containing 2<br/>

CAB39  -  calcium binding protein 39<br/>

RGS10  -  regulator of g-protein signaling 10<br/>

SERPINB1  -  serpin peptidase inhibitor, clade b (ovalbumin), member 1<br/>

FNIP2  -  folliculin interacting protein 2<br/>

RGS2  -  regulator of g-protein signaling 2, 24kda<br/>

RGS1  -  regulator of g-protein signaling 1<br/>

PPP1R10  -  protein phosphatase 1, regulatory subunit 10<br/>

CCNL1  -  cyclin l1<br/>

STK4  -  serine/threonine kinase 4<br/>

PPP1R11  -  protein phosphatase 1, regulatory (inhibitor) subunit 11<br/>

CTSA  -  cathepsin a<br/>

CCNT1  -  cyclin t1<br/>

CCNY  -  cyclin y<br/>

EID1  -  ep300 interacting inhibitor of differentiation 1<br/>

PPME1  -  protein phosphatase methylesterase 1<br/>

FNIP1  -  folliculin interacting protein 1<br/>

PHACTR1  -  phosphatase and actin regulator 1<br/>

STYXL1  -  serine/threonine/tyrosine interacting-like 1<br/>

ROCK1  -  rho-associated, coiled-coil containing protein kinase 1<br/>

SERPINB8  -  serpin peptidase inhibitor, clade b (ovalbumin), member 8<br/>

SERPINB9  -  serpin peptidase inhibitor, clade b (ovalbumin), member 9<br/>

KIDINS220  -  kinase d-interacting substrate, 220kda<br/>

CXCL10  -  chemokine (c-x-c motif) ligand 10<br/>

ANXA4  -  annexin a4<br/>

ADPRH  -  adp-ribosylarginine hydrolase<br/>

TAOK3  -  tao kinase 3<br/>

RUNDC1  -  run domain containing 1<br/>

SHOC2  -  soc-2 suppressor of clear homolog (c. elegans)<br/>

PRKAR1A  -  protein kinase, camp-dependent, regulatory, type i, alpha<br/>

PKIG  -  protein kinase (camp-dependent, catalytic) inhibitor gamma<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0046961&amp;view=details" target="_blank">GO:0046961</a><a name="0046961"></a></td>
<td>proton-transporting ATPase activity, rotational mechanism</td>
<td align="center">3.92E-4</td>
<td align="center">7.21E-1</td>
<td align="center">5.70 (8795,18,514,6)</td>
<td>
<a href="javascript:toggle('elements_GO:0046961')" id="ctrl_elements_GO:0046961">[+] Show genes</a><br/>
<div id="elements_GO:0046961" style="display:none"> <font size="1">

ATP6V1G1  -  atpase, h+ transporting, lysosomal 13kda, v1 subunit g1<br/>

ATP6V1E1  -  atpase, h+ transporting, lysosomal 31kda, v1 subunit e1<br/>

ATP6V0D1  -  atpase, h+ transporting, lysosomal 38kda, v0 subunit d1<br/>

ATP6V0E1  -  atpase, h+ transporting, lysosomal 9kda, v0 subunit e1<br/>

ATP6V1D  -  atpase, h+ transporting, lysosomal 34kda, v1 subunit d<br/>

ATP6V0C  -  atpase, h+ transporting, lysosomal 16kda, v0 subunit c<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0042030&amp;view=details" target="_blank">GO:0042030</a><a name="0042030"></a></td>
<td>ATPase inhibitor activity</td>
<td align="center">7.59E-4</td>
<td align="center">9.3E-1</td>
<td align="center">12.83 (8795,4,514,3)</td>
<td>
<a href="javascript:toggle('elements_GO:0042030')" id="ctrl_elements_GO:0042030">[+] Show genes</a><br/>
<div id="elements_GO:0042030" style="display:none"> <font size="1">

FNIP2  -  folliculin interacting protein 2<br/>

ATPIF1  -  atpase inhibitory factor 1<br/>

FNIP1  -  folliculin interacting protein 1<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0019888&amp;view=details" target="_blank">GO:0019888</a><a name="0019888"></a></td>
<td>protein phosphatase regulator activity</td>
<td align="center">8.51E-4</td>
<td align="center">7.81E-1</td>
<td align="center">3.23 (8795,53,514,10)</td>
<td>
<a href="javascript:toggle('elements_GO:0019888')" id="ctrl_elements_GO:0019888">[+] Show genes</a><br/>
<div id="elements_GO:0019888" style="display:none"> <font size="1">

PHACTR4  -  phosphatase and actin regulator 4<br/>

PPP1R10  -  protein phosphatase 1, regulatory subunit 10<br/>

PPP2R2A  -  protein phosphatase 2, regulatory subunit b, alpha<br/>

PPP4R2  -  protein phosphatase 4, regulatory subunit 2<br/>

SHOC2  -  soc-2 suppressor of clear homolog (c. elegans)<br/>

PPME1  -  protein phosphatase methylesterase 1<br/>

PHACTR1  -  phosphatase and actin regulator 1<br/>

ANKLE2  -  ankyrin repeat and lem domain containing 2<br/>

STYXL1  -  serine/threonine/tyrosine interacting-like 1<br/>

PPP1R11  -  protein phosphatase 1, regulatory (inhibitor) subunit 11<br/>
</font></div>
</td>
</tr>
<tr>
<td><a href="http://www.godatabase.org/cgi-bin/amigo/go.cgi?query=GO:0036442&amp;view=details" target="_blank">GO:0036442</a><a name="0036442"></a></td>
<td>proton-exporting ATPase activity</td>
<td align="center">9.87E-4</td>
<td align="center">7.25E-1</td>
<td align="center">4.89 (8795,21,514,6)</td>
<td>
<a href="javascript:toggle('elements_GO:0036442')" id="ctrl_elements_GO:0036442">[+] Show genes</a><br/>
<div id="elements_GO:0036442" style="display:none"> <font size="1">

ATP6V1G1  -  atpase, h+ transporting, lysosomal 13kda, v1 subunit g1<br/>

ATP6V1E1  -  atpase, h+ transporting, lysosomal 31kda, v1 subunit e1<br/>

ATP6V0D1  -  atpase, h+ transporting, lysosomal 38kda, v0 subunit d1<br/>

ATP6V0E1  -  atpase, h+ transporting, lysosomal 9kda, v0 subunit e1<br/>

ATP6V1D  -  atpase, h+ transporting, lysosomal 34kda, v1 subunit d<br/>

ATP6V0C  -  atpase, h+ transporting, lysosomal 16kda, v0 subunit c<br/>
</font></div>
</td>
</tr>
</table>
<p>Species used: Homo sapiens</p><p>The system has recognized 9558 genes out of 10021 gene terms entered by the user.<br/> 9558 genes were recognized by gene symbol and 0 genes by other gene IDs .<br/>530 duplicate genes were removed (keeping the highest ranking instance of each gene) leaving  a total of 9028 genes.<br/>Only 8795 of these genes are associated with a GO term.<br/></p>
<br/><a href="GOFUNCTION.xls"> Output in Microsoft Excel format</a><br/>
<p> The GOrilla database is periodically updated using the <a href="http://www.geneontology.org/">GO database</a> and other sources.<br/>

The GOrilla database was last updated on Mar 6, 2021</p>
<p>This results page will be available on this site for one month from now (until 

Jan 16, 2023

). You can bookmark this page and come back to it later.

</p>
<a name="p_value_info"></a><br/><b>'P-value'</b> is the enrichment p-value computed according to the mHG or HG model. This p-value is not corrected for multiple testing of 3672 GO terms.<br/>
<a name="fdr_info"></a><br/><b>'FDR q-value'</b> is the correction of the above p-value for multiple testing using the Benjamini and Hochberg (1995) method. <br/>Namely, for the i<sup>th</sup> term (ranked according to p-value) the FDR q-value is (p-value * number of GO terms) / i. <br/>
<a name="enrich_info"></a><br/><b>Enrichment (N, B, n, b)</b> is defined as follows:<br/>N - is the total number of genes<br/>B - is the total number of genes associated with a specific GO term<br/>n - is the number of genes in the top of the user's input list or in the target set when appropriate<br/>b - is the number of genes in the intersection<br/>Enrichment = (b/n) / (B/N)<br/>
<a name="genes_info"></a><br/><b>Genes:</b> For each GO term you can see the list of associated genes that appear in the optimal top of the list.<br/>Each gene name is specified by gene symbol followed by a short description of the gene <br/>
<p>
<a href="http://cbl-gorilla.cs.technion.ac.il/"> Back to the GOrilla main page </a>
</p>
</body>
</html>
